projects/packages/videopress/src/client/block-editor/hooks/use-video-data-update/index.ts

```diff
@@ -209,12 +204,14 @@ export function useSyncMedia(
 						],
 					} );
 				} );
-			}
-		}
 
-		// Re-render video player
-		const videoPressUrl = getVideoPressUrl( guid, attributes );
-		invalidateResolution( 'getEmbedPreview', [ videoPressUrl ] );
+				const videoPressUrl = getVideoPressUrl( guid, attributes );
+				invalidateResolution( 'getEmbedPreview', [ videoPressUrl ] );
+			} else {
+				const videoPressUrl = getVideoPressUrl( guid, attributes );
+				invalidateResolution( 'getEmbedPreview', [ videoPressUrl ] );
+			}
```

Comment:
No need for the `else` here.

```suggestion
			} 

			const videoPressUrl = getVideoPressUrl( guid, attributes );
			invalidateResolution( 'getEmbedPreview', [ videoPressUrl ] );
```

Reply:
this is an async process so I had to move the invalidation part 8f7679d9e202a4bca185733914e220ede54f6d1e